ci-operator/step-registry/openshift/microshift/includes/openshift-microshift-includes-commands.sh

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env bash
 
+export EXIT_CODE_AWS_EC2_FAILURE=3
+export EXIT_CODE_AWS_EC2_LOG_FAILURE=4
+export EXIT_CODE_LVM_INSTALL_FAILURE=5
+export EXIT_CODE_RPM_INSTALL_FAILURE=6
+export EXIT_CODE_CONFORMANCE_SETUP_FAILURE=7
+export EXIT_CODE_PCP_FAILURE=8
+export EXIT_CODE_WAIT_CLUSTER_FAILURE=9
```

Comment:
These seem to only be used in one script. Are we planning to share the exit codes? 
If not, maybe they belong to that one script?

Reply:
Each of these codes is used at least in 2 scripts. One, the step they belong to, and secondly the sos report step which will generate additional junit files (just like must-gather step does in ocp). If there is some code only used once then I missed some step.